routes/utils.js

```diff
@@ -65,10 +171,29 @@ function updateCurrentProjectFiles(config, token, session, project, callback) {
   });
 }
 
+function getRemixedProjectFiles(config, projectId, callback) {
+  var publishURL = config.publishURL + "/publishedProjects/" + projectId + "/publishedFiles";
+
+  request.get({ uri: publishURL }, function(err, response, body) {
+    if(err) {
+      console.error("Failed to send request to " + publishURL + " with: ", err);
+      callback(null, 500);
```

Comment:
Shouldn't this not be null? It could help to tell a request failure apart from a publish error.